ml-dhs-aws/src/main/java/com/marklogic/client/dhs/CountInputStream.java

```diff
@@ -28,7 +28,7 @@ public int read() throws IOException {
 		int readLength = super.read();
 		if (readLength > 0)
 			byteCount += readLength;
-		return readLength;
+		return addToByteCount(super.read());
```

Comment:
The suggestion was to delete lines 28 - 30 as well.  At present, the revision reads two bytes and counts two bytes.